docs/contributing.rst

```diff
@@ -39,6 +39,7 @@ Check out the code and run ``vagrant up``::
 
     $ git clone https://github.com/fedora-infra/noggin
     $ cd noggin
+    $ mkdir ../freeipa-fas
```

Comment:
Ah, this made me notice a bug in the Vagrantfile. I fixed it, you can remove this line, thanks.